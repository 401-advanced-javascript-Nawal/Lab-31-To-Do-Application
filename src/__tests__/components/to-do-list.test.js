import React from 'react';
import { shallow, mount } from 'enzyme';
import renderer from 'react-test-renderer';
import ToDoList from '../../modules/to-do-list.js';


describe ('<ToDoList />' , () => {

  it(' It Should be exists in our applications ', () => {
    // virtual 
    let app = shallow(<ToDoList />);
    // find the tag which already includes in our app 
    expect(app.find('p').exists()).toBeTruthy();
  }); // end of it test 

  it(' Renders correctly ' , () => {
    const tree = renderer.create(<ToDoList />).toJSON();
    expect(tree).toMatchSnapshot();
  }); // end of it test for render 

}); // end of describe test 