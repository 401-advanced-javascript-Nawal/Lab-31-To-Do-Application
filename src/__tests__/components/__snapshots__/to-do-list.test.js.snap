// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ToDoList />  Renders correctly  1`] = `
<section>
  <form
    onSubmit={[Function]}
  >
    <p>
       Enter Item To Do List :
      <input
        onChange={[Function]}
      />
    </p>
    <p>
       Set Difficulty ( from 1 to 5 ) :
      <input
        className="dif"
        max="5"
        min="1"
        type="range"
      />
    </p>
  </form>
  <p>
     Items :
  </p>
</section>
`;
